healthcheck/ticker.go

```diff
@@ -0,0 +1,68 @@
+package healthcheck
+
+import (
+	"context"
+	"github.com/ONSdigital/go-ns/log"
+	"time"
+)
+
+type ticker struct {
```

Comment:
I can see why this type is unexported - it should certainly only be accessed by this healthcheck package - but I would expect that to also mean its' fields would be unexported. 

Because the type is unexported, anything that uses it wouldn't be able to access its type and know about its fields - so there's no point in its' fields being exported. 

And anywhere this type is used (like in the `HealthCheck struct { Tickers []*ticker}` I would expect its use to be unexported - we wouldnt want an external package which has a `HealthCheck` to do anything with `Tickers`

Reply:
Think this boiled down to me not realising it the struct was Pascal case then it is public rather than private, likewise that if the fields/properties were camalCase they would be private/unexported.